python/pypto/jit/specializer.py

```
@@ -0,0 +1,926 @@
+            and isinstance(node.targets[0], ast.Tuple)
+            and isinstance(node.value, ast.Attribute)
+            and node.value.attr == "shape"
+            and isinstance(node.value.value, ast.Name)
+            and node.value.value.id in self._meta
+        ):
+            param_name = node.value.value.id
+            meta = self._meta[param_name]
+            target_names = node.targets[0].elts
+            if len(target_names) == len(meta.shape):
+                stmts: list[ast.stmt] = []
+                for tgt, (i, dim) in zip(target_names, enumerate(meta.shape)):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)